funnel/assets/js/register_form_modal.js

```diff
@@ -0,0 +1,32 @@
+import Vue from 'vue/dist/vue.esm';
+import jsonForm from './utils/jsonform';
+
+Vue.config.devtools = true;
+
+const FormUI = {
+  init(jsonSchema) {
+    /* eslint-disable no-new */
+    new Vue({
+      el: '#register-form',
+      data() {
+        return {
+          jsonSchema,
+        };
+      },
+      components: {
+        jsonForm,
+      },
+      methods: {
+        onChange(event) {
+          this.data = event.data;
+        },
+      },
+    });
+  },
+};
+
+$(() => {
+  window.Hasgeek.registerForm = (jsonSchema) => {
```

Comment:
The term "register" is also shared with account creation. Maybe that should be renamed to "login" in all internal references, if there are any left.